applications/ContactStructuralMechanicsApplication/custom_strategies/custom_strategies/residualbased_newton_raphson_contact_strategy.h

```diff
@@ -103,7 +103,7 @@ class ResidualBasedNewtonRaphsonContactStrategy :
     
     typedef ModelPart::ConditionsContainerType                                ConditionsArrayType;
     
-    typedef boost::shared_ptr<ProcessFactoryUtility>                            ProcessesListType;
+    typedef ProcessFactoryUtility::Pointer                                      ProcessesListType;
```

Comment:
shouldn't this be `Kratos::shared_ptr<ProcessFactoryUtility>`?

Reply:
@philbucher it is exactly the same type (and both are kratos-specific, so no reason to prefer one over the other).